catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceSource.java

```diff
@@ -124,12 +123,18 @@ public ConfluenceSource(FilterAdapter adapter, EncryptionService encryptionServi
     }
 
     public void init() {
+        if(StringUtils.isBlank(endpointUrl)){
```

Comment:
❓ do we have a unit test that hits this condition. If not it would be nice to add to prevent any regressions there.